testdata/config/metrics.yaml

```diff
@@ -12,6 +12,7 @@ spec:
     method: request.path | "unknown"
     version: target.labels["version"] | "unknown"
     response_code: response.code | 200
+  monitored_resource_type: '"UNSPECIFIED"'
```

Comment:
@mandarjog is it possible to submit change to this file separately first (force push if you want). This change is blocking manual testing of mixer using the testdata configs (for the report templates only).

@douglas-reid FYI, since he ran into this issue last night.